applications/production.py

```diff
@@ -243,19 +241,18 @@ def main():
                 shower.printResourcesReport()
 
     # ArraySimulators
-    arraySimulators = dict()
-    for primary, configData in arrayConfigs.items():
-        aa = ArraySimulator(label=label, configData=configData)
-        arraySimulators[primary] = aa
-
     if not args.showers_only:
+        arraySimulators = dict()
+        for primary, configData in arrayConfigs.items():
+            aa = ArraySimulator(label=label, configData=configData)
```

Comment:
Why is this split into two lines? I actually saw you fixed this somewhere, just can't remember if it was here or somewhere else.

Reply:
No idea - ` it wasn't me`

Reply:
OK but this can definitely be made into one line then.